### DISCUSSION TITLE
bwrealm.net down

### DISCUSSION THREAD

**Discussion by hartmark** [openblack/openblack#803] [Show and tell]
I just saw that the 0.1.0 release went out and wanted to try it out.

It seems bwrealm.net is down.

I was however able to get the patch 1.1 and 1.2 from here:
* https://download.cnet.com/black-white-1-1-patch/3000-2121_4-10236146.html
* https://download.cnet.com/black-white-1-2-clan-patch/3000-2121_4-10236148.html

Otherwise it seems to be working well, 800 FPS on my machine xD

Keep up the good work, it will be interesting to follow the progress of this project.

**Comment by bwrsandman** [comment]
Try https://www.bwgame.net/
